tests/test_opendssdirect.py

```diff
@@ -4694,7 +4694,7 @@ def test_linegeometry_class_to_dataframe():
             "cond": "4",
             "wire": "ACSR1/0",
             "x": [0.0, -1.2909, -0.1530096, 0.5737],
-            "h": [13.716, 13.716, 4.1806368, 13.716],
+            "h": [14.648, 13.716, 4.1806368, 13.716],
```

Comment:
@PMeira I moved to the single loop and the results that I get also change. The single loop is correct, but I'm not sure why the previous one didn't work.